hooks/014-set-motd.chroot

```diff
@@ -23,3 +23,6 @@ rm /etc/update-motd.d/10-help-text
 
 # remove the motd-news service files
 rm /lib/systemd/system/motd-news.{service,timer}
+
+# diable dynamic motd
+sed -i 's/ENABLED=1/ENABLE=0/g' /etc/default/motd-news
```

Comment:
ENABLED=0, not ENABLE=0.

This makes sense, to avoid the network hit to poll for content that won't be displayed, as part of the console login.

Reply:
Oops, it's a ridiculous mistake, I'll fix it.